vibi-dpu/src/github/auth.rs

```diff
@@ -20,9 +20,8 @@ struct Claims {
 }
 
 fn generate_jwt(github_app_id: &str) -> Result<String, Box<dyn std::error::Error>> {
```

Comment:
We will not be returning error from this function. We will return an Option<String>. If there is an error, log it in this function and then return none, otherwise return the jwt string

Reply:
right. made the changes.